src/NSubstitute/AssemblyProperties.cs

```diff
@@ -0,0 +1,6 @@
+﻿using System.Runtime.CompilerServices;
+using Castle.Core.Internal;
+
+// Make internal types visible to Castle's dynamic assembly as we might define want
```

Comment:
minor typo: "might define want to" -> "might want to". Can fix later, but felt like I should contribute _something_ to this review 😂

Reply:
Fixed! Well, now I know that you at least review them before approving 🤣

Reply:
Just testing huh? 😆😜